packages/SwingSet/src/types-external.js

```diff
@@ -185,6 +185,10 @@ export {};
  * @typedef {object} SwingSetOptions
  * @property {string} [bootstrap]
  * @property {ConfigProposal[]} [coreProposals]
+ * @property {string[]} [exportStorageSubtrees] chain storage paths
+ *   for which data should be exported into bootstrap vat parameter `chainStorageEntries`
+ *   (e.g., `exportStorageSubtrees: ['a.b']` might result in vatParameters including
+ *   `chainStorageEntries: [ ['a.b', ''], ['a.b.c', '"foo"'], ['a.b.c2', '42'] ]`).
```

Comment:
So I just checked the `"entries"` implementation and it's something I should have foreseen, and I think I made a mistake in #6971.

An entry that is just a placeholder should not have a value. We should have `['a.b']` instead of `['a.b', '']` for placeholder ancestors.

Reply:
Updated to match "get"; empty non-terminals now look like `[path, null]`.